doc/source/whatsnew/v0.23.0.rst

```diff
@@ -1,7 +1,9 @@
 .. _whatsnew_0230:
 
-v0.23.0 (May 15, 2018)
-----------------------
+What's new in 0.23.0 (May 15, 2018)
```

Comment:
Not a big deal, but "new" is capitalized for all the others. I'd go with lower-case for all.